frontend/public/components/monitoring.tsx

```diff
@@ -1197,37 +1194,45 @@ const QueryBrowserPage = withFallback(() => {
                 <button type="submit" className="btn btn-primary">Run Queries</button>
               </div>
             </div>
-            {_.map(queries, (q, i) => <div className="group" key={i}>
-              <div className="group__title">
-                <button
-                  className="btn btn-link query-browser__table-toggle-btn"
-                  onClick={() => toggleExpanded(i)}
-                  title={`${q.expanded ? 'Hide' : 'Show'} Table`}
-                >
-                  <i
-                    aria-hidden="true"
-                    className={`fa fa-angle-${q.expanded ? 'down' : 'right'} query-browser__table-toggle-icon`}
+            {_.map(queries, (q, i) => {
+              const toggleEnabledLabel = `${q.enabled ? 'Disable' : 'Enable'} query`;
+              return <div className="group" key={i}>
+                <div className="group__title">
+                  <button
+                    className="btn btn-link query-browser__table-toggle-btn"
+                    onClick={() => toggleExpanded(i)}
+                    title={`${q.expanded ? 'Hide' : 'Show'} Table`}
+                  >
+                    <i
+                      aria-hidden="true"
+                      className={`fa fa-angle-${q.expanded ? 'down' : 'right'} query-browser__table-toggle-icon`}
+                    />
+                  </button>
+                  <input
+                    className="form-control query-browser__query"
+                    onBlur={e => onQueryBlur(e, i)}
+                    onChange={e => onQueryChange(e, i)}
+                    placeholder="Expression (press Shift+Enter for newlines)"
+                    type="text"
+                    value={q.text}
                   />
-                </button>
-                <input
-                  className="form-control query-browser__query"
-                  onBlur={e => onQueryBlur(e, i)}
-                  onChange={e => onQueryChange(e, i)}
-                  placeholder="Expression (press Shift+Enter for newlines)"
-                  type="text"
-                  value={q.text}
-                />
-                <div className="dropdown-kebab-pf query-browser__kebab">
-                  <Kebab options={[
-                    {label: `${q.enabled ? 'Disable' : 'Enable'} query`, callback: () => toggleEnabled(i)},
-                    {label: 'Delete query', callback: () => deleteQuery(i)},
-                  ]} />
+                  <div className="query-browser__query-switch">
+                    <Tooltip content={toggleEnabledLabel}>
```

Comment:
I have a feeling this means you'll have to tap the switch twice on iOS, once for the tooltip and once for the switch.

@cshinn can we consider removing the tooltip?

Reply:
@spadgett yeah, I don't think it's totally necessary. Its function should be immediately obvious once you click on it and there's no real cost for clicking it accidentally.

Reply:
I tested on iOS and it actually works with a single tap. However, after discussing with @cshinn, we decided to remove the tooltip anyway since it's not really necessary and it could potentially cause touch screen issues.

Reply:
Removed by https://github.com/openshift/console/pull/1792